.github/workflows/publish_arm.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish ARM
+
+on:
+  workflow_run:
+    workflows: ["Full Clients Publish"]
+    types:
+      - completed
+
+jobs:
+  docker-image-grid-http-server:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: 'Get tag'
+        id: tag
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            let run = await github.rest.actions.getWorkflowRun({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: context.payload.workflow_run.id,
+            });
+            return run.data.head_branch;
+
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ steps.tag.outputs.result }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
why do we use `qemu` and `buildx` here?

Reply:
Some libraries that we depend on use native binaries that are not prepackaged for ARM and thus need to be compiled. Building under emulation with QEMU is a convenient way to do this without needing to bother with cross compiling.